samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -326,8 +326,10 @@ - (UIImage*)transformImage:(UIImage*)image {
     // create CGImageRef from vImage_Buffer output
     CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
 
-    CGContextRef ctx = CGBitmapContextCreate(
-        outBuffer.data, outBuffer.width, outBuffer.height, 8, outBuffer.rowBytes, colorSpace, (CGBitmapInfo)kCGImageAlphaNoneSkipLast);
+    const CGBitmapInfo bitmapInfo = kCGImageAlphaNoneSkipLast | kCGBitmapByteOrder32Little;
```

Comment:
BitmapInfo is something apps should already be specifying when creating contexts or images.